include/crm/common/alerts_internal.h

```diff
@@ -30,6 +30,8 @@ typedef struct {
     int timeout;
     char *tstamp_format;
     char *recipient;
+    char *select_kind;
+    char *select_attribute_name;
```

Comment:
I considered the output to log and prepared the field which I handled in g_strsplit and a field of the original character string.